modules/dnn/test/test_layers.cpp

```diff
@@ -618,6 +618,94 @@ TEST(Layer_LSTM_Test_Accuracy_with_, HiddenParams)
     normAssert(h_t_reference, outputs[0]);
 }
 
+TEST(Layer_GATHER_1d_Test, Accuracy) {
+
+    LayerParams lp;
+    lp.type = "Gather";
+    lp.name = "gatherLayer";
+    lp.set("axis", 1);
+    lp.set("real_ndims", 1);
+
+    Ptr<GatherLayer> layer = GatherLayer::create(lp);
+
+    cv::Mat input = cv::Mat::ones(1, 1, CV_32F);
+    cv::Mat indices = cv::Mat::ones(1, 1, CV_32F) * 0;
+    cv::Mat output_ref = cv::Mat::ones(1, 1, CV_32F);
+
+    std::vector<Mat> inputs{input, indices};
+    std::vector<Mat> outputs;
+
+    runLayer(layer, inputs, outputs);
+    ASSERT_EQ(shape(output_ref), shape(outputs[0]));
+    normAssert(output_ref, outputs[0]);
+}
+
+TEST(Layer_ARGMIN_1d_Test, Accuracy) {
+
+    LayerParams lp;
+    lp.type = "Arg";
+    lp.name = "argLayer";
+    lp.set("op", "min");
+    lp.set("axis", 1);
+    lp.set("keepdims", 1);
+    lp.set("select_last_index", 0);
+
+    Ptr<ArgLayer> layer = ArgLayer::create(lp);
+
+    cv::Mat input = cv::Mat::ones(1, 1, CV_32F);
+    cv::Mat output_ref = cv::Mat::ones(1, 1, CV_32F) * 0;
+
+    std::vector<Mat> inputs{input};
+    std::vector<Mat> outputs;
+
+    runLayer(layer, inputs, outputs);
+    ASSERT_EQ(shape(output_ref), shape(outputs[0]));
+    normAssert(output_ref, outputs[0]);
+}
+
+typedef testing::TestWithParam<tuple<int, std::string>> Layer_ADD_1d_Test;
+TEST_P(Layer_ADD_1d_Test, Accuracy) {
+
+    int batch_size = get<0>(GetParam());
+    std::string operation = get<1>(GetParam());
+
+    LayerParams lp;
+    lp.type = "Eltwise";
+    lp.name = "addLayer";
+    lp.set("operation", operation);
+    Ptr<EltwiseLayer> layer = EltwiseLayer::create(lp);
+
+    std::vector<int> input_shape = {batch_size, 1, 1};
+    if (batch_size == 0)
+        input_shape.erase(input_shape.begin());
+
+    cv::Mat input1 = cv::Mat(input_shape, CV_32F, 1.0);
+    cv::Mat input2 = cv::Mat(input_shape, CV_32F, 1.0);
+
+    for (int i = 0; i < batch_size; ++i) {
+        input1.at<float>(i, 0, 0) = static_cast<float>(i + 1);
```

Comment:
.at<float>(i)

Reply:
This triggers the debug assert when it is nd mat (n>=2). See

https://github.com/opencv/opencv/blob/49367b5bec57a0b135db4c2270604c14f5a5dc31/modules/core/include/opencv2/core/mat.inl.hpp#L938-L940